line_detection_updated.py

```diff
@@ -0,0 +1,302 @@
+        # Outliers on the positive side take the value of the cut.
+        # Outliers on the negative side take the value 0
+        up_limit = trimmed_image.mean() + self.nstd1_cut*trimmed_image.std()
+        low_limit = trimmed_image.mean() - self.nstd1_cut*trimmed_image.std()
+        trimmed_image[trimmed_image > up_limit] = up_limit
+        trimmed_image[trimmed_image <= low_limit] = 0
+
+        # Standardizing the image and moving it away from 0
+        processed_image = (trimmed_image-trimmed_image.mean())/trimmed_image.std()
+        processed_image -= processed_image.min()
+
+        # Fitting the image to a 0-255 range
+        processed_image = (255*(processed_image - processed_image.min())
+                           / (processed_image.max() - processed_image.min()))
+
+        # Thresholding the processed image
+        limit = processed_image.mean() + self.nstd2_binary_cut*processed_image.std()
+        threshold, thresholded_image = cv2.threshold(processed_image, limit, 255, cv2.THRESH_BINARY)
+        thresholded_image = cv2.convertScaleAbs(thresholded_image)
+
+        # Masking image to remove any borders
+        if self.mask is True:
+            x_top, x_bottom, y_left, y_right = image_mask(thresholded_image, self.mask_percent)
+
+            thresholded_image = thresholded_image[x_top:x_bottom, y_left:y_right]
+
+        # Deciding Kernel size of blur based on amount of noise
+        num_bright_pixels = np.sum(thresholded_image > np.mean(thresholded_image))
+        x = thresholded_image.shape[0]
+        y = thresholded_image.shape[1]
+        prop_bright_pixels = num_bright_pixels/(x*y)
+
+        for i in range(len(self.flux_prop_thresholds)):
+            if prop_bright_pixels < self.flux_prop_thresholds[i]:
+                kernel_blur = self.blur_kernel_sizes[i]
+                break
+
+        # Blur the image
+        blurred_image = cv2.medianBlur(thresholded_image, kernel_blur)
+
+        # Performing Canny edge detection
+        edges = cv2.Canny(blurred_image, 0, 200)
+
+        return (thresholded_image, blurred_image, edges)
+
+    def hough_transformation(self):
+        '''
+        Detects the straight lines in an image
+
+        Parameters
+        ----------
+        self : 'class'
+            The LineDetection class to use its __init__ parameters
+
+        Returns
+        -------
+        lines: 'numpy.ndarray'
+            The polar coordinates of detected lines
+        angles_list: 'list'
+            List of angles (in radians) for each detected line
+        cart_coords_list: 'list'
+            List of endpoint cartesian coordinates for each detected line
+        thresholded_image: 'numpy.ndarray'
+            The thresholded image from process_image returns
+        blurred_image: 'numpy.ndarray'
+            The blurred image from process_image returns
+        edges: 'numpy.ndarray'
+            The Canny edge detected image from process_image returns
```

Comment:
Same as above, spaces between parameter names and backticks.

Reply:
I understand the backticks comment, not sure what the spaces comment means? I have converted the single quotes to backticks throughout